contributions/catalog/webapps/templates/style.html

```diff
@@ -0,0 +1,156 @@
+<style>
```

Comment:
I think we will have to move this to a CSS file in a future task.

Reply:
yes. I agree. There are several things that needs to be organized.